### PR TITLE
Integrate GitHub Actions

### PR DESCRIPTION
**Pull request by calumshep**
Scope to add PHPUnit testing for CI/CD

### CODE REVIEW

File: .github/workflows/deploy-dev.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.PUSH_TOKEN }}
+
+    - name: Deploy to production
+      uses: appleboy/ssh-action@master
+      with:
+        username: laravel
+        host: 161.35.32.131
```
Comment (hcphoon01) [inline .github/workflows/deploy-dev.yml:31]: Could this be put in an env variable

File: .github/workflows/deploy-master.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.PUSH_TOKEN }}
+
+    - name: Deploy to production
+      uses: appleboy/ssh-action@master
+      with:
+        username: laravel
+        host: 161.35.32.131
```
Comment (hcphoon01) [inline .github/workflows/deploy-master.yml:31]: Same as previous comment

File: .github/workflows/deploy-dev.yml (hunk 1 above)
Comment (calumshep) [reply .github/workflows/deploy-dev.yml:31]: The hostname? Don't think it's necessary as the server IP is publicly visible anyway. Okay it's hidden by the domain but it's not exactly difficult to expose the actual server IP.

Comment (hcphoon01) [reply .github/workflows/deploy-dev.yml:31]: More for if it needs to be changed for any reason

**Pull request merged**